blockchain/vm/gas.go

```diff
@@ -38,18 +37,16 @@ const (
 
 // calcGas returns the actual gas cost of the call.
 //
-// The cost of gas was changed during the homestead price change HF. To allow for EIP150
-// to be implemented. The returned gas is gas - base * 63 / 64.
-func callGas(gasTable params.GasTable, availableGas, base uint64, callCost *big.Int) (uint64, error) {
-	if gasTable.CreateBySuicide > 0 {
-		availableGas = availableGas - base
-		gas := availableGas - availableGas/64
-		// If the bit length exceeds 64 bit we know that the newly calculated "gas" for EIP150
-		// is smaller than the requested amount. Therefor we return the new gas instead
-		// of returning an error.
-		if !callCost.IsUint64() || gas < callCost.Uint64() {
-			return gas, nil
-		}
+// The returned gas is gas - base * 63 / 64.
+
+func callGas(availableGas, base uint64, callCost *big.Int) (uint64, error) {
+	availableGas = availableGas - base
+	gas := availableGas - availableGas/64
+	// If the bit length exceeds 64 bit we know that the newly calculated "gas" for EIP150
+	// is smaller than the requested amount. Therefor we return the new gas instead
+	// of returning an error.
+	if !callCost.IsUint64() || gas < callCost.Uint64() {
+		return gas, nil
 	}
 	if !callCost.IsUint64() {
```

Comment:
The same condition is being checked twice.

Reply:
umm.. you're right. 
original code is this. https://github.com/ethereum/go-ethereum/blob/6ec15610443b28eabf665199f1dc5be2b3e3f7cb/core/vm/gas.go#L37-L53.
This code is executed when the node's hardfork version is lower than eip150. However, klaytn's node can only have constantinople version, so this code is useless. I will remove this code.